node/src/accountant/db_access_objects/sent_payable_dao.rs

```diff
@@ -502,39 +515,20 @@ mod tests {
             .initialize(&home_dir, DbInitializationConfig::test_default())
             .unwrap();
         let subject = SentPayableDaoReal::new(wrapped_conn);
-        let tx1 = TxBuilder::default()
-            .hash(H256::from_low_u64_le(1))
-            .status(TxStatus::Pending)
-            .build();
+        let tx1 = TxBuilder::default().hash(H256::from_low_u64_le(1)).build();
         let tx2 = TxBuilder::default()
             .hash(H256::from_low_u64_le(2))
-            .status(TxStatus::Failed)
-            .build();
-        let tx3 = TxBuilder::default()
-            .hash(H256::from_low_u64_le(3))
-            .status(TxStatus::Succeeded(TransactionBlock {
-                block_hash: Default::default(),
-                block_number: Default::default(),
-            }))
-            .build();
-        let tx4 = TxBuilder::default()
-            .hash(H256::from_low_u64_le(4))
-            .status(TxStatus::Pending)
-            .build();
-        let tx5 = TxBuilder::default()
-            .hash(H256::from_low_u64_le(5))
-            .status(TxStatus::Failed)
+            .block(Default::default())
             .build();
+        let tx3 = TxBuilder::default().hash(H256::from_low_u64_le(3)).build();
```

Comment:
Please change the places where you construct the H256 so that it uses our utility function `make_tx_hash()`.
I believe that the big amount of lines where you do this can be substituted that way. 

Here, particularly, it would be 
```Rust
make_tx_hash(3).
```